src/components/performances/PerformancesSection.jsx

```diff
@@ -37,29 +39,33 @@ class PerformancesSection extends React.Component {
   }
 
   render() {
-    const headings = ['Dance Title', 'Dance Entry', 'School', 'Acaademic Level', 'Level of Competition', 'Dance Style', 'Dance Size'];
     const { performances } = this.state;
     const { match: { params: { eventId }}} = this.props;
-    const collectionName = `events/${eventId}/performances`;
+    const headings = ['Dance Title', 'Dance Entry', 'School', 'Acaademic Level', 'Level of Competition', 'Dance Style', 'Dance Size'];
+    const showPerformances = Array.isArray(performances) && performances.length > 0;
 
     return (
       <React.Fragment>
         <SectionHeader eventId={eventId} title="performance" />
+        <TableSettings />
         <Table>
           <TableHeader headings={headings} />
           <TableBody>
-            {(Array.isArray(performances) && performances.length)
-              && performances.map(performance => (
+            {showPerformances && performances.map((performance) => {
+              const keys = ['danceEntry', 'danceTitle', 'performers', 'danceStyle', 'competitionLevel', 'choreographers', 'academicLevel', 'school', 'size'];
+              const currentValues = pick(performance, keys);
+              return (
                 <PerformanceTableRow
-                  key={performance.id}
-                  {...performance}
+                  currentValues={currentValues}
                   eventId={eventId}
-                  collectionName={collectionName} />
-              ))}
+                  id={performance.id}
```

Comment:
Because performance.id is used more than once, I'd almost recommend we just destructure the performance object for this id and only pass in id. Though i'd lean either way, what are your thoughts @livictoria